packages/next/package.json

```
@@ -11,9 +11,9 @@
     "url": "https://github.com/fostyfost/redux-eggs/issues"
   },
   "homepage": "https://github.com/fostyfost/redux-eggs",
-  "main": "dist/index.server.js",
-  "module": "dist/index.server.js",
-  "browser": "dist/index.client.js",
+  "main": "dist/index.server.es.js",
```

```suggestion
  "main": "dist/index.server.js",
```

this should be the cjs bundle I think